### ISSUE TITLE
Spliting the PDF into single pages -during do_ocr 2nd, 3rd, 4th.....run

### ISSUE THREAD

**Issue by jayantanth** [tshrinivasan/OCR4wikisource#37]
I have observed that every re-run do_ocr to complete the full OCR, every time this script Spliting the PDF into single pages. Although all single present at folder. Can it be eliminate this repeated process?

**Comment by tshrinivasan** [comment]
Yet to think on that.

I hope apart from there is a little time delay, this is not a big issue.

If you want still, will plan for this in few days.

**Comment by jayantanth** [comment]
Agreed! Take your time and if possible fixed it.  this is not a big issue
